src/App.js

```diff
@@ -1,23 +1,61 @@
-import logo from './logo.svg';
+
+
+
+import {useState, useEffect} from "react";
+import axios from "axios" ;
 import './App.css';
 
+
 function App() {
+  // setting up const variables for the app
```

Comment:
Nice job creating comments to show what your block of code is executing! Very helpful!